airflow/metrics/statsd_logger.py

```diff
@@ -44,27 +46,32 @@
 log = logging.getLogger(__name__)
 
 
-def prepare_stat_with_tags(fn: T) -> T:
-    """Add tags to stat with influxdb standard format if influxdb_tags_enabled is True."""
+def prepare_metric_name_with_tags(fn: T) -> T:
+    """Add tags to metric_name with InfluxDB standard format if influxdb_tags_enabled is True."""
 
     @wraps(fn)
     def wrapper(
-        self, stat: str | None = None, *args, tags: dict[str, str] | None = None, **kwargs
+        self, metric_name: str | None = None, tags: dict[str, str] | None = None
```

Comment:
Nit: you changed tags from keyword only to positional with this change.

```suggestion
        self, metric_name: str | None = None, *, tags: dict[str, str] | None = None
```

Reply:
This also possibly breaks things as `stats.incr("metric_a", 2). Please bring `*args` back.

Reply:
"prepare_metric_name_with_tags()" is only being called before "get_name()" which is only being called in the functions. Also when I added them back it broke some of the previously written test cases